### DISCUSSION TITLE
I cant use github education benefits

### DISCUSSION THREAD

**Discussion by alper-dipci** [community/community#137597] [GitHub Education]
### Hi everyone,

I approved at 31 august but still didnt get my approval mail . Do i need to do something ? I cant use any github student benefits

**Comment by andrefqms** [comment]
It usually takes 1-3 business days to receive the benefits of the GitHub Student Developer Pack after being approved. However, it may take longer depending on the volume of applications that GitHub is receiving. 

They've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

But I suggest that you check your email inbox (including the spam folder) for any communication from GitHub regarding your application. If you can't find any, you should contact GitHub Support directly. They will be able to provide you with more information about your application status. You can reach out to them through this [link](https://support.github.com/contact)

**Comment by davevad93** [comment]
Hi @alper-dipci ,

Don't worry, you don't have to do anything, just wait; when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @alper-dipci was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
